client/src/Components/HomePage/MostPopularItems/MostPopularItems.jsx

```diff
@@ -53,6 +58,7 @@ export default function MostPopularItems() {
             </SwiperSlide>
           ))}
       </Swiper>
+      <Login isLoginDialogOpen={isOpenDialog} />
```

Comment:
It's a bit strange that you render Login component in MostPopularItems component. I guess it should be rendered in some common place, like header or footer. Seems you will not have MostPopularItems component on other pages except Home, but you still need Login form there